app/scripts/metamask-controller.js

```diff
@@ -5798,33 +5790,6 @@ export default class MetamaskController extends EventEmitter {
     }
   };
 
-  async securityProviderRequest(requestData, methodName) {
```

Comment:
I'm finding remnants of `securityProviderRequest`. Should the other ones be removed as well? 
- /app/scripts/controllers/mmi-controller.test.ts
- /app/scripts/lib/createRPCMethodTrackingMiddleware.js
- /app/scripts/lib/createRPCMethodTrackingMiddleware.test.js